wiki/Beatmap_information/en.md

```diff
@@ -0,0 +1,141 @@
+
+Further below these action buttons from left to right may be present, allowing the following:
+
+- **Favourite:** Add this beatmap to the favourites section in the user profile.
+- **Download:** Download the beatmap. If the beatmap has a background video, the user can choose to download the beatmap with or without the video.
+- **osu!direct**: Download the beatmap directly in the game client without requiring to manually open the file.
+- **Report:** If the beatmap is in either the [graveyard](/wiki/Beatmap/Category#graveyard), or the [WIP and the pending](/wiki/Beatmap/Category#wip-and-pending) category, the user has the ability to [report the beatmap](/wiki/Reporting_bad_behaviour#beatmap), should there be any rule-violating content. By clicking on the 3 dots, a dropdown menu appears, revealing the `report` button.
+
+### Statistics panel
+
+::: Infobox
+![](img/statistics-panel.png "Statistics panel showing the beatmap settings")
+:::
+
+To the right side of the difficulty menu is the statistics panel. Above the panel, the beatmap's category can be seen. Small icons will indicate if the beatmap has a video or a storyboard. When clicking the triangle-shaped button below the category, a short preview of the song will be played. Pause the preview by clicking a second time.
+
+Underneath the preview menu the song length, the BPM, as well as the number of [hit objects](/wiki/Gameplay/Hit_object) can be seen from left to right respectively. Upon hovering over the song length, a tooltip will additionally show the [drain](/wiki/Beatmap/Drain_time) length.
+
+Depending on the game mode, the following beatmap settings may be visible, along with the corresponding value next to it:
+
+| Setting | Description | Game mode |
+| :-: | :-- | :-: |
+| [Circle size](/wiki/Beatmap/Circle_size) (CS) | Determines how big the hit circles are. | ![][osu!] ![][osu!catch] |
+| [HP drain](/wiki/Beatmap/HP_drain_rate) (HP) | Defines how much health is gained and lost while playing. | ![][osu!] ![][osu!taiko] ![][osu!catch] ![][osu!mania] |
+| [Approach rate](/wiki/Beatmap/Approach_rate) (AR) | Regulates the speed hit objects appear in. | ![][osu!] ![][osu!catch] |
+| [Accuracy](/wiki/Gameplay/Accuracy) | Measures how good the player hits hit objects on time. | ![][osu!] ![][osu!taiko] ![][osu!catch] ![][osu!mania] |
+| Key count | Specifies the number of keys used when playing a beatmap. | ![][osu!mania] |
+
+At the bottom of the statistics panel is the [star rating](/wiki/Beatmap/Star_rating) detailing how much skill the player requires for the beatmap.
+
+::: Infobox
+![](img/user-rating.png "User rating")
+:::
+
+If the beatmap is [qualified](/wiki/Beatmap/Category#qualified), [ranked](/wiki/Beatmap/Category#ranked), or [loved](/wiki/Beatmap/Category#loved), a user rating will be displayed below the statistics panel. User rating is a feature in osu!(stable) where after passing a beatmap users are able to vote between 1 and 10 stars based on how much they liked it.
```

Comment:
Capitalised when making reference to the category. Uncapitalised if making reference to the verb (to rank, to qualify, etc...).
```suggestion
If the beatmap is [Qualified](/wiki/Beatmap/Category#qualified), [Ranked](/wiki/Beatmap/Category#ranked), or [Loved](/wiki/Beatmap/Category#loved), the user rating will be displayed below the statistics panel. After passing a beatmap in osu!(stable), users are able to vote between 1 and 10 stars based on how much they liked it.
```